### DISCUSSION TITLE
Re-add deleted device

### DISCUSSION THREAD

**Discussion by Dellle** [SukramJ/aiohomematic#1747] [Q&A] [answered]
Hi everyone,
I deleted a device from HA integration. It is still available with Rasperrymatic. Is there a way to re-add that device in HA?

Regards
Sebastian

**Comment by SukramJ** [comment, marked as answer]
Restart HA?

**Reply by Dellle** [reply]
Yes thanks 🤦, a complete restart brought the device back.
